tests/phpunit/tests/block-bindings/register.php

```diff
@@ -72,15 +87,16 @@ public function test_get_all_registered() {
 	 *
 	 * @covers ::register_block_bindings_source
 	 * @covers ::unregister_block_bindings_source
+	 * @covers WP_Block_Bindings_Source::__construct
 	 */
 	public function test_unregister_block_source() {
-		register_block_bindings_source( self::TEST_SOURCE_NAME, self::TEST_SOURCE_PROPERTIES );
+		register_block_bindings_source( self::$test_source_name, self::$test_source_properties );
 
-		$result = unregister_block_bindings_source( self::TEST_SOURCE_NAME );
-		$this->assertSame(
-			array_merge(
-				array( 'name' => self::TEST_SOURCE_NAME ),
-				self::TEST_SOURCE_PROPERTIES
+		$result = unregister_block_bindings_source( self::$test_source_name );
+		$this->assertEquals(
```

Comment:
Changed `assertSame` to `assertEqual` in the tests because we're now comparing two object instances instead of two arrays.